SM6115 vendor ADSP firmware registers APM on GPR module IID 3 and PRM
on IID 7, instead of the mainline defaults of 1 and 2. Change the
defines to match the vendor firmware.
---
 include/dt-bindings/soc/qcom,gpr.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/dt-bindings/soc/qcom,gpr.h b/include/dt-bindings/soc/qcom,gpr.h
--- a/include/dt-bindings/soc/qcom,gpr.h
@@ -12,8 +12,8 @@
 /* Static Services */

-#define GPR_APM_MODULE_IID		1
-#define GPR_PRM_MODULE_IID		2
+#define GPR_APM_MODULE_IID		3
+#define GPR_PRM_MODULE_IID		7
 #define GPR_AMDB_MODULE_IID		3
 #define GPR_VCPM_MODULE_IID		4

